### ISSUE TITLE
Videos are small

### ISSUE THREAD

**Issue by nikhilCad** [FredJul/Flym#722]
This is reddit feed video in the flym app
![Screenshot_20201111-101755](https://user-images.githubusercontent.com/47214394/98770107-7880a580-2407-11eb-9749-123fd03e518a.png)

And this is the same reddit feed video in an app called Handy News Reader
![Screenshot_20201111-101819](https://user-images.githubusercontent.com/47214394/98770150-93ebb080-2407-11eb-8646-4fbc1dbfcb4a.png)

The one in Flym is too small to see. Please fix this.

**Comment by nikhilCad** [comment]
This ia the only little annoyance with this app otherwose its amazing

**Comment by xGian18** [comment]
Yeah also u can't put full screen videos because the option is disabled.

![IMG_20201114_132843](https://user-images.githubusercontent.com/51527160/99147008-850e3380-267d-11eb-9966-d9c48b1e82ef.jpg)
